samza-core/src/main/scala/org/apache/samza/job/JobRunner.scala

```diff
@@ -118,6 +118,28 @@ class JobRunner(config: Config) extends Logging {
     }
     coordinatorSystemProducer.stop()
 
+
+    // if diagnostics is enabled, create diagnostics stream if it doesnt exist
+    if (new JobConfig(config).getDiagnosticsEnabled) {
+      val DIAGNOSTICS_STREAM_ID = "samza-diagnostics-stream-id"
+      val systemStreamName = new MetricsConfig(config).
```

Comment:
Is this just a stream, not a systemStream? Also, could you please add a prefix like "diagnostics" to make it more specific to this block?